docs/fips.html

```diff
@@ -27,6 +27,18 @@
 	    (revision 2.0.16).
 	    </p>
 
+            <p>
+            Neither validation will work with any release other than 1.0.2.
+            The OpenSSL project is no longer maintaining either the 1747
+            or the 2398 module. This includes adding platforms to those
+            validations.
+            We are starting work on a new validation, based on the upcoming
+            1.1.1 release.
```

Comment:
Technically "after the 1.1.1 release completes" is the better wording. 

This is not a 1.1.1 focused effort - it is for the next release after 1.1.1 which has not yet been labelled. I'm concerned that people will read that to mean we are committing to a 1.1.1 release FIPS solution which are **not** doing so.